src/commands/android/index.ts

```diff
@@ -325,41 +309,6 @@ export class AndroidSetup {
     return answers.javaHome;
   }
 
-  getSdkRootFromEnv(): string {
-    Logger.log('Checking the value of ANDROID_HOME environment variable...');
-
-    const androidHome = process.env.ANDROID_HOME;
-    const fromDotEnv = this.otherInfo.androidHomeInGlobalEnv ? '' : ' (taken from .env)';
-
-    if (androidHome) {
-      const androidHomeFinal = untildify(androidHome);
-
```

Comment:
`untildify` is no longer used in this file. Remove all unused imports.

Reply:
It is still used by the function `getSdkRootFromUser`. I don't think there are any unused imports.